SPECS/librdkafka/librdkafka.spec

```diff
@@ -17,11 +13,15 @@ Group:          Development/Libraries/C and C++
 URL:            https://github.com/edenhill/librdkafka
 #Source0:        https://github.com/edenhill/%{name}/archive/v%{version}.tar.gz
 Source0:        %{name}-%{version}.tar.gz
+%global debug_package %{nil}
```

Comment:
Could we keep the macros at the top of the spec? It's what we do for most specs and I think they are easier to find this way. I think a linter might use a small tweak here to be honest.:)